target/linux/ramips/dts/mt7620a_tplink_re2x0-v1.dtsi

```diff
@@ -63,9 +63,11 @@
 			};
 
 			partition@20000 {
-				compatible = "tplink,firmware";
+				compatible = "openwrt,uimage", "denx,uimage";
 				label = "firmware";
 				reg = <0x20000 0x7c0000>;
+				openwrt,offset = <0x1000>;
+				openwrt,ih-magic = <IH_MAGIC_OKLI>;
```

Comment:
CI reports the following error.

```
Error: ../dts/mt7620a_tplink_re2x0-v1.dtsi:70.25-26 syntax error
FATAL ERROR: Unable to parse input tree
```

`#include <dt-bindings/mtd/partitions/uimage.h>` needs to be added to use `IH_MAGIC_OKLI`.

Reply:
Fixed. I restored the necessary change when I commit the final patch.